vision/unit_tests/test_image.py

```diff
@@ -36,26 +36,26 @@ def _make_one(self, *args, **kw):
     def test_image_source_type_content(self):
         image = self._make_one(CLIENT_MOCK, content=IMAGE_CONTENT)
 
-        _AS_DICT = {
+        _as_dict = {
             'content': B64_IMAGE_CONTENT
         }
 
         self.assertEqual(B64_IMAGE_CONTENT, image.content)
         self.assertEqual(None, image.source)
-        self.assertEqual(_AS_DICT, image.as_dict())
+        self.assertEqual(image.as_dict(), _as_dict)
 
     def test_image_source_type_google_cloud_storage(self):
         image = self._make_one(CLIENT_MOCK, source_uri=IMAGE_SOURCE)
 
-        _AS_DICT = {
+        _as_dict = {
```

Comment:
Also, why `_as_dict` instead of `as_dict`? Who are we hiding from in a unit test?